examples/add_tag.yml

```diff
@@ -10,8 +10,8 @@
       resource_name: openshift_provider
       state: present
       tags:
-      - {category: environment, name: qa}
-      - {category: department, name: accounting}
+      - category: environment, name: qa
+      - category: department, name: accounting
```

Comment:
Thanks but I'm afraid this comma is not legal YAML (ansible complains).  I believe it should be:
```
       tags:
       - category: environment
         name: qa
       - category: department
         name: accounting
```